test/Quest.spec.ts

```diff
@@ -95,6 +95,7 @@ describe('Erc20Quest', async () => {
       protocolFeeRecipient.address,
       deployedErc20Quest.address,
       owner.address,
+      owner.address, // as a placeholder, would the QuestTerminalKey contract
```

Comment:
Should we also have `deployQuestTerminalKey` in this spec file as well in order to update `questTerminalKeyContract` variable?

[nit] Maybe zero address would be a better placeholder